gradle-plugin/src/main/kotlin/com/google/devtools/ksp/gradle/AndroidPluginIntegration.kt

```diff
@@ -81,11 +81,14 @@ object AndroidPluginIntegration {
     ) {
         if (androidComponent != null && project.isAgpBuiltInKotlinUsed()) {
             if (project.canUseInternalKspApis()) {
-                val sources =
+                val javaSources =
                     (androidComponent.sources.java as? FlatSourceDirectoriesForJavaImpl)?.allButKspAndKaptGenerators()
 
+                val kotlinSources = androidComponent.sources.kotlin?.static
```

Comment:
nit: Add a TODO(bug) to consider `kotlin.all` or `kotlin.allExceptKspOutputs` later so we don't forget?

(I believe we need `kotlin.allExceptKspOutputs`, but if you think `kotlin.all` is safe then perhaps we can confirm that with a test?)